yarn.lock

```diff
@@ -1848,6 +2113,19 @@
     "@smithy/types" "^2.5.0"
     tslib "^2.5.0"
 
+"@aws-sdk/credential-provider-sso@3.540.0":
+  version "3.540.0"
```

Comment:
I guess these are fine? ideally the changes to `yarn.lock` would reflect only the diff in `package.json`, but node dependency tooling is funny